.travis.yml

```diff
@@ -1,44 +1,22 @@
 language: python
+
 python:
   - "3.6"
+  - "3.7"
+  - "3.8"
+  - "3.9"
```

Comment:
Maybe skip 3.9 for now, seems not yet compatible with Galaxy -- or the other way round :)